### ISSUE TITLE
ETTh2 training result

### ISSUE THREAD

**Issue by Alsac** [romilbert/samformer#9]
![image](https://github.com/user-attachments/assets/f2189a58-c005-48b3-8a08-2753df1912f2)
![image](https://github.com/user-attachments/assets/9214dd48-7eb1-4d5b-a4d0-865a5c4c868f)

Thank you for your great job! I first test the pytorch code run_demo.py for ETTh2 data when pred_len=336, and the test set RMSE: 0.7076962253299964; MSE: 0.5008339473463251, which not as good as the paper. I wondering if I not set the parameter properly?
